source/includes/steps-auto-schema-generation.yaml

```
@@ -0,0 +1,94 @@
+title: Define a Realm Schema
+ref: define-realm-schema
+content: |
+
+  To get started, ensure you have a Realm Schema defined. This Realm
+  Schema will be translated into a Realm Object Model to be configured
+  and utilized in your mobile application.
+
+  .. note:: 
+
+    To learn how to define a Realm Schema, see the :doc:`Enforce a Document Schema </mongodb/enforce-a-document-schema>` page.
+
+---
+title: View the Realm Object Model
+ref: view-realm-db-schema
+content: |
+
+  The Realm Object Model will be used to define and validate your data
```

Something like: (avoid future tense and passive voice)
```suggestion
  The Realm Object Model defines and validates your data
```